db/migrate/20160425233022_add_attachment_avatar_to_companies.rb

```diff
@@ -0,0 +1,11 @@
+class AddAttachmentAvatarToCompanies < ActiveRecord::Migration
+  def self.up
+    change_table :companies do |t|
+      t.attachment :avatar
```

Comment:
@LevinGeorge Can you please change the name from avatar to logo. Avatar doesn't seem to make sense for their logo.